04_numeronym/NimishMudhale/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronym(vals ...string) []string {
+	numeronymArray := make([]string, len(vals))
+	for i, val := range vals {
+		target := strings.Trim(val, "\n\t")
```

Comment:
should Trim space as well: strings.Trim(val, "\n\t ")

Reply:
done